.circleci/config.yml

```diff
@@ -44,7 +44,7 @@ jobs:
           command: make test_debian9_image_size
       - run:
           name: Running debian9 CI Tests
```

Comment:
Not that important, but should rename this as well

Reply:
yup I noticed that. Let's change this after this release since we are short on time for this release.